### DISCUSSION TITLE
How can I perform the simulation in dynamic wave?

### DISCUSSION THREAD

**Discussion by alejorojas95** [pyswmm/pyswmm#542] [General]
I have an inp file that when simulated normally generates an error but when generating the simulation in `dynamic wave` it runs fine, as would the configuration in pyswmm, thanks.

![onda dinamica](https://github.com/pyswmm/pyswmm/assets/57581430/1403a38f-8b6b-4ac5-a94c-ca821e13ee42)

**Comment by bemcdonnell** [comment]
@alejorojas95, Are you saying that when you try running with Kinematic Wave it does not run?  And when you select Dynamic Wave, it runs?

**Reply by alejorojas95** [reply]
> @alejorojas95, Are you saying that when you try running with Kinematic Wave it does not run? And when you select Dynamic Wave, it runs?

but epa swmm run with dynamic wave and it works since the network was configured so it would be possible to run it in pyswmm that type dynamic wave

**Reply by bemcdonnell** [reply]
@alejorojas95 i am not understanding what you are asking.  If the model runs with epa SWMM using dynamic wave, it should also run with pyswmm using dynamic wave. What versions of SWMM and pyswmm are you using?
